src/components/Gesture.svelte

```diff
@@ -140,7 +166,8 @@
       (buttons.buttonA && triggerButton === MicrobitInteractions.A) ||
       (buttons.buttonB && triggerButton === MicrobitInteractions.B)
     )
-      recordClicked();
+    showCountdown = true;
```

Comment:
You can move all of the `showCountdown = true` into the top of the `countdownStart` function, so it is only needs to be set in one place.